download_and_create_reference_datasets/v02/hail_scripts/write_combined_reference_data_ht.py

```diff
@@ -217,6 +229,28 @@
 CONFIG['dbnsfp_mito']['38']['filter'] = lambda ht: ht.locus.contig == 'chrM'
 
 
+def annotate_ref_regions(ht, region_dataset, reference_genome):
+    """
+    Annotate combined reference data with region based reference data.
+    :param ht: Combined reference hail Table.
+    :param region_dataset: Imported region/bed file with 3 or 4 columns
+    :param reference_genome: Reference genome, e.g. GRCh38
```

Comment:
The `e.g. GRCh38` is a little confusing. It could be more clear to change it to `e.g. '38' for GRCh38`